<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-2F9I86H" ignored="0" name="TestOnChrome" tests="5" failures="4" timestamp="2020-04-07T12:32:40 IST" time="24.115" errors="0">
  <testcase name="@BeforeClass launchBrowser" time="16.977" classname="TestCases.LoginTestCase">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at base.TestBase.open_Browser(TestBase.java:69)
at TestCases.LoginTestCase.launchBrowser(LoginTestCase.java:33)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeClass launchBrowser -->
  <testcase name="fbLoginTest" time="0.0" classname="TestCases.LoginTestCase">
    <skipped/>
  </testcase> <!-- fbLoginTest -->
  <testcase name="clickAdminLink" time="0.006" classname="TestCases.HomeTestCase">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at base.TestBase.click_On_Button(TestBase.java:172)
at pageObjects.HomePage.homeDashBoard(HomePage.java:27)
at TestCases.HomeTestCase.clickAdminLink(HomeTestCase.java:32)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- clickAdminLink -->
  <testcase name="add_usersTestCase" time="2.005" classname="TestCases.HomeTestCase">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at TestCases.HomeTestCase.add_usersTestCase(HomeTestCase.java:41)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- add_usersTestCase -->
  <testcase name="delUsersTestCase" time="0.019" classname="TestCases.HomeTestCase">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at TestCases.HomeTestCase.delUsersTestCase(HomeTestCase.java:49)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- delUsersTestCase -->
  <testcase name="logOutTc" time="5.003" classname="TestCases.HomeTestCase">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at base.TestBase.click_On_Button(TestBase.java:172)
at pageObjects.HomePage.logOutFunc(HomePage.java:20)
at TestCases.HomeTestCase.logOutTc(HomeTestCase.java:55)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- logOutTc -->
</testsuite> <!-- TestOnChrome -->
